inventory/all_projects/_orphans

```diff
@@ -5,7 +5,6 @@ issuetracker-prod1.princeton.edu
 lib-adc1.princeton.edu
 lib-adc2.princeton.edu
 [monitor_backends]
```

Comment:
We should probably remove the group as well:
```suggestion
```

Reply:
Okay, we'll remove it and do another commit on this branch when we come back to this ticket tomorrow. Thanks @acozine, I wasn't sure if we would need this group later.

Reply:
This group was removed